src/components/Aside/AsideList.svelte

```diff
@@ -8,16 +9,25 @@
 	import AsideCoinList from './AsideCoinList.svelte';
 
 	const pages = [
-		{ name: '홈', component: HomeIcon },
-		{ name: '공지사항', component: NoticeIcon },
-		{ name: '거래', component: TradeIcon },
-		{ name: '거래내역', component: TradeRecordIcon },
-		{ name: '지갑', component: WalletIcon }
+		{ name: '홈', icon: HomeIcon },
+		{ name: '공지사항', icon: NoticeIcon },
+		{ name: '거래', icon: TradeIcon },
+		{ name: '거래내역', icon: TradeRecordIcon },
+		{ name: '지갑', icon: WalletIcon }
 	];
 
+	const pagesMap: Record<string, string> = {
+		홈: '/',
+		공지사항: '/notice',
+		거래: '/trade',
+		거래내역: '/trade/history',
+		지갑: '/wallet'
+	};
```

Comment:
배열이나 Record중 하나만 사용하는 방법은 어떨까요?
코드라인도 줄일 수 있고 자료형을 하나 더 선언하지 않아도 되어 메모리 관점으로 보았을 때도
더욱 효율적일 것 같다고 생각합니다!

```suggestion
const pages = [
    { name: "홈", icon: HomeIcon, href: "/" },
    ...
]
```

Reply:
그렇네요!